### PR TITLE
Update vb6-support.md

### PR DESCRIPTION
**Pull request by ghost**
# Title
Update vb6-support.md to remove text saying "Custom Support Agreements may be available".

## Summary
Update vb6-support.md to remove text saying "Custom Support Agreements may be available".

## Details
Custom Support Agreements are not available for Visual Basic 6 IDE.

## Suggested Reviewers
paulyuk@microsoft.com

### CODE REVIEW

**Comment by mairaw** [comment]
@paulyuk you were requested as a reviewer here. Can you take a look? Thx!

**Review by mairaw — COMMENTED** [review]
LGTM. It would be good to change the date on line 3 to reflect this topic has been updated recently.

**Review by paulyuk — APPROVED** [review]

File: docs/visual-basic/reference/vb6-support.md [hunk 1]
```diff
@@ -14,21 +14,21 @@ ms.assetid: ffc5ba4d-44d7-4ef7-a3f6-38a8738bf127
 author: "paulyuk"
 ms.author: "paulyuk"
 ---
-# Support Statement for Visual Basic 6.0 on Windows Vista, Windows Server 2008, Windows 7, Windows 8 and Windows 8.1, Windows Server 2012, Windows 10, and Windows Server 2016
+# Support Statement for Visual Basic 6.0 on Windows
 
-Update: March 2016
```
Comment (svick) [inline docs/visual-basic/reference/vb6-support.md]: Is it necessary to specify the update date twice? Currently, the start of the article looks like this:

Comment (mairaw) [reply docs/visual-basic/reference/vb6-support.md]: No, we should remove this line

**Comment by svick** [comment]
This article doesn't seem to be correctly placed in the TOC. Is that intentional?

**Comment by mairaw** [comment]
@svick I've noticed that too. Including @rpetrusha here since he brought the topic over and maybe knows whether it is by design or not.

**Comment by rpetrusha** [comment]
@svick @mairaw Yes, it's by design. Since it's a legacy topic, it's available only from links on support topics.

File: docs/visual-basic/reference/vb6-support.md (hunk 1 above)
Comment (danru-ms) [reply docs/visual-basic/reference/vb6-support.md]: Thanks @svick @mairaw!  Removed in latest commit to the PR.

**Comment by danru-ms** [comment]
@mairaw I believe this is ready to pull.  LMK if you have any other feedback.

**Review by mairaw — APPROVED** [review]
LGTM 👍

**Pull request merged**